atom.gyp

```diff
@@ -271,8 +275,14 @@
               '-lcomctl32.lib',
               '-lcomdlg32.lib',
               '-lwininet.lib',
+              '-lwinmm.lib',
             ],
           },
+          'defines': [
+            # The usage of "webrtc/modules/desktop_capture/desktop_capture_options.h"
+            # is required to see this macro.
+            'WEBRTC_WIN',
```

Comment:
Can you move this and `WEBRTC_MAC` to brightray? We usually put Chromium's defines there.